plugin/__tests__/fixtures/buildGradleFiles.ts

```diff
@@ -19,7 +19,8 @@ android {
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 34
         versionName "1.16.2"
-        buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()`;
+        buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()
+        buildConfigField "boolean", "ENABLE_AMAZON_DRM", getExtOrDefault("enableAmazonDrm")`;
```

Comment:
Could we make this look more like a boolean? Something like `isAmazonDrmEnabled` would be great.